tools/ncbi_blast_plus/ncbi_macros.xml

```diff
@@ -381,11 +589,41 @@
 #end if
     </token>
     <token name="@ADV_WORD_SIZE@">
-#if (str($adv_opts.word_size) and int(str($adv_opts.word_size)) > 0):
+#if str($adv_opts.word_size) != "":
```

Comment:
This can probably be further simplified or made more Pythonic is you put:

```
#if str($adv_opts.word_size):
```

(This is what you've done in the ``@ADV_MATRIX_GAPCOSTS@`` token macro)